Tests/colorLib/builder_test.py

```diff
@@ -223,105 +223,132 @@ def test_buildCPAL_invalid_color():
 
 
 def test_buildColorIndex():
-    c = builder.buildColorIndex(0)
-    assert c.PaletteIndex == 0
+    c = builder._build(ot.ColorIndex, 1)
+    assert c.PaletteIndex == 1
     assert c.Alpha.value == 1.0
     assert c.Alpha.varIdx == 0
 
-    c = builder.buildColorIndex(1, alpha=0.5)
+
+def test_buildColorIndex_Alpha():
+    c = builder._build(ot.ColorIndex, (1, 0.5))
     assert c.PaletteIndex == 1
     assert c.Alpha.value == 0.5
     assert c.Alpha.varIdx == 0
 
-    c = builder.buildColorIndex(3, alpha=builder.VariableFloat(0.5, varIdx=2))
+
+def test_buildColorIndex_Variable():
+    c = builder._build(ot.ColorIndex, (3, builder.VariableFloat(0.5, varIdx=2)))
     assert c.PaletteIndex == 3
     assert c.Alpha.value == 0.5
     assert c.Alpha.varIdx == 2
 
 
 def test_buildPaintSolid():
-    p = LayerV1ListBuilder().buildPaintSolid(0)
+    p = builder._buildPaint((ot.PaintFormat.PaintSolid, 0))
     assert p.Format == ot.PaintFormat.PaintSolid
     assert p.Color.PaletteIndex == 0
     assert p.Color.Alpha.value == 1.0
     assert p.Color.Alpha.varIdx == 0
 
-    p = LayerV1ListBuilder().buildPaintSolid(1, alpha=0.5)
+
+def test_buildPaintSolid_Alpha():
+    p = builder._buildPaint((ot.PaintFormat.PaintSolid, (1, 0.5)))
     assert p.Format == ot.PaintFormat.PaintSolid
     assert p.Color.PaletteIndex == 1
     assert p.Color.Alpha.value == 0.5
     assert p.Color.Alpha.varIdx == 0
 
-    p = LayerV1ListBuilder().buildPaintSolid(
-        3, alpha=builder.VariableFloat(0.5, varIdx=2)
-    )
+
+def test_buildPaintSolid_Variable():
+    p = builder._buildPaint((2, (3, builder.VariableFloat(0.5, varIdx=2))))
```

Comment:
maybe the use of int literal here is intentional? It may break if we change again format numbers
```suggestion
    p = builder._buildPaint((ot.PaintFormat.PaintSolid, (3, builder.VariableFloat(0.5, varIdx=2))))
```

Reply:
I just missed it :) I have some tests that use int(ot.PaintFormat.Whatever) to confirm we handle ints